xarray/core/options.py

```diff
@@ -41,6 +45,17 @@ def _set_file_cache_maxsize(value):
 }
 
 
+def _get_keep_attrs(default):
+    global_choice = OPTIONS['keep_attrs']
+
+    if global_choice is 'default':
+        return default
+    elif global_choice in [True, False]:
+        return global_choice
+    else:
+        raise ValueError('The global option keep_attrs is set to an invalid value.')
```

Comment:
maybe say "The global option keep_attrs must be one of True, False or 'default'."?